src/python/grpcio/grpc/__init__.py

```diff
@@ -1705,13 +1777,14 @@ def composite_channel_credentials(channel_credentials, *call_credentials):
             channel_credentials._credentials))
 
 
-def ssl_server_credentials(private_key_certificate_chain_pairs,
-                           root_certificates=None,
-                           require_client_auth=False):
+def ssl_server_credentials(
+        private_key_certificate_chain_pairs: Sequence[Sequence],
+        root_certificates: Optional[bytes] = None,
+        require_client_auth: Optional[bool] = False) -> ServerCredentials:
     """Creates a ServerCredentials for use with an SSL-enabled Server.
 
     Args:
-      private_key_certificate_chain_pairs: A list of pairs of the form
+      private_key_certificate_chain_pairs: A sequence of pairs of the form
```

Comment:
Changed it to `Sequence` since some users are using [sequence](https://github.com/gaia-pipeline/pythonsdk/blob/c4d25740f038ed6e9cffcf68d5dde9ddc2a4f804/gaiasdk/sdk.py#L142).